cmd/frontend/graphqlbackend/webhooks.go

```diff
@@ -74,6 +75,43 @@ func (r *schemaResolver) Webhooks(ctx context.Context, args *struct {
 	return &webhookConnectionResolver{}
 }
 
+func (r *schemaResolver) Webhook(ctx context.Context, args *struct {
+	ID   *graphql.ID
+	UUID *string
+}) (*webhookResolver, error) {
+	if err := auth.CheckCurrentUserIsSiteAdmin(ctx, r.db); err != nil {
+		return nil, errors.New("must be site admin")
+	}
+
+	if (args.ID == nil && args.UUID == nil) || (args.ID != nil && args.UUID != nil) {
```

Comment:
```suggestion
	if (args.ID == nil && args.UUID == uuid.nil) || (args.ID != nil && args.UUID != uuid.Nil) {
```

`UUID` is never nil by itself, its zero value is exposed as `uuid.Nil`

Reply:
That's the arg itself, which is an optional String, and is passed as `nil` by the GraphQL resolver